### PR TITLE
Fix/chatbot attachments

### LINKED ISSUE #319: Attachments for chatbot
Each time an image is sent, a duplicate file is created (they are identical), e.g. 
```
media/original_images/1a982068-3664-4649-86f0-226d3f40f91c.png
media/original_images/1a982068-3664-4649-86f0-226d3f40f91c_omyTYP5.png
media/original_images/1a982068-3664-4649-86f0-226d3f40f91c_xoiCh1j.png
```
This doesn't seem to happen for other file types.

If an image URL is split across messages, the image is not recognized as such.
Before PR #312:
image
After:
image
Note: the length of the text itself is 1159 chars, so the second attachment URL is the one that gets split.

You can test this out by pressing the first button here: http://iogt-django-dev.idems.international/en/sections/youth/internet-safety/i-spam/

### PR DESCRIPTION
**Pull request by ehmadzubair**
Fixes #319

### CODE REVIEW

File: messaging/api/views.py [hunk 1]
```diff
@@ -19,6 +21,7 @@ class RapidProWebhook(APIView):
 
     def post(self, request):
         serializer = RapidProMessageSerializer(data=request.data)
+        print(json.dumps(request.data))
```
Comment (geoo89) [inline messaging/api/views.py:24]: Remove debug code?

File: docker-compose.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ services:
       WAGTAILTRANSFER_SOURCE_BASE_URL: https://example.com/wagtail-transfer/
       WAGTAILTRANSFER_SOURCE_SECRET_KEY: wagtailtransfer-source-secret-key
       BASE_URL: 'http://localhost:8000'
-    command: bash -c "python manage.py reload_po_files && python manage.py compilemessages && python manage.py runserver 0.0.0.0:8000"
```
Comment (geoo89) [inline docker-compose.yml:24]: Is there a reason that these have been removed, or is this for testing only?

**Review by geoo89 — COMMENTED** [review]
Just a quick look at the code, will do actual testing in a bit.

**Comment by geoo89** [comment]
Apart from these two issues everything is working fine!

**Comment by sheralim012** [comment]
@geoo89 changes addressed

**Pull request merged**